packages/saasify-utils/lib/annotate-openapi.js

```diff
@@ -34,7 +34,7 @@ module.exports = async (spec, deployment, opts = { }) => {
 
   if (readme) {
     readme = processReadme(readme)
-    readme = `# Readme\n\n${readme}`
+    readme = `# README\n\n${readme}`
```

Comment:
not sure i like this change; in general, the whole all caps readme thing is a little dated. sindre started using lowercase like all other file names and i've followed suit.

could possibly be convinced otherwise in this case though I feel like `Readme` looks cleaner.